Update TSDB-comparison doc and `Merge` command doc (#3233)
apache/iotdb: docs/UserGuide/Comparison/TSDB-Comparison.md

10 clients read data concurrently. The number of storage group is 10. There are 10 devices and each device has 10 measurements (i.e.,, 100 time series total).
The data type is *double*, encoding type is *GORILLA*

* Compression:

> We use several famous dataset as test. We compare TsFile(the file format of IoTDB) with Parquet, ORC and Csv by their file size after the same dataset is written.

Reviewer: ```suggestion
We test and compare file sizes of TsFile(the file format of IoTDB) and some others famous dataset formats, which are Parquet, ORC and Csv, after the same datasets are written.
```

![Aggregation query](https://user-images.githubusercontent.com/24886743/106251336-cf03c000-624f-11eb-8395-de5e349f47b5.png)

> <center>Figure 6. Aggregation query time cost(ms) IoTDB v0.11.1</center>

Reviewer: Where is the Figure 5?
